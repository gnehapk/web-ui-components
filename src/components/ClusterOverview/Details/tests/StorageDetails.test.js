import React from 'react';
import { render } from 'enzyme';

import { StorageDetails } from '../StorageDetails';
import { default as StorageDetailsFixtures } from '../fixtures/StorageDetails.fixture';

const testStorageDetailsOverview = () => <StorageDetails {...StorageDetailsFixtures[0].props} />;

describe('<StorageDetails />', () => {
  it('renders correctly', () => {
    const component = render(testStorageDetailsOverview());
    expect(component).toMatchSnapshot();
  });
});
