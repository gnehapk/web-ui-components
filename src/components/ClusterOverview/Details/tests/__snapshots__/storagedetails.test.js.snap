// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StorageDetails /> renders correctly 1`] = `
<div
  class="card-pf kubevirt-dashboard__card"
>
  <div
    class="card-pf-heading kubevirt-dashboard__card-heading"
  >
    <h2
      class="card-pf-title"
    >
      OCS Details
    </h2>
  </div>
  <div
    class="card-pf-body"
  >
    <dl>
      <dt
        class="kubevirt-detail__item-title"
      >
        Name
      </dt>
      <dd
        class="kubevirt-detail__item-value"
      >
        <div>
          <div
            class="spinner spinner-md blank-slate-pf-icon"
          />
        </div>
      </dd>
      <dt
        class="kubevirt-detail__item-title"
      >
        Provider
      </dt>
      <dd
        class="kubevirt-detail__item-value"
      >
        <span>
          Ceph
        </span>
      </dd>
      <dt
        class="kubevirt-detail__item-title"
      >
        OCS version
      </dt>
      <dd
        class="kubevirt-detail__item-value"
      >
        <span>
          v4.2
        </span>
      </dd>
    </dl>
  </div>
</div>
`;
